### DISCUSSION TITLE
Github action virtual display resolution for windows server 2019

### DISCUSSION THREAD

**Discussion by souravd000** [community/community#49112] [Actions] [answered]
### Select Topic Area

Question

### Body

Is there any way to set virtual resolution for windows server 2019 to 1920x1080 in the workflow.yml file?

**Comment by jsoref** [comment, marked as answer]
Probably. https://github.com/actions/runner-images/issues/2935#issuecomment-799516736

**Reply by souravd000** [reply]
Thanks

**Reply by jsoref** [reply]
If this was the answer, please accept it as the answer to help the next reader.

**Reply by souravd000** [reply]
Sure
